content/apps/using-openid.md

```diff
@@ -0,0 +1,122 @@
+# Using OpenID Connect to authenticate users
+
+::: warning
+**This feature is available for our partners using an Akeneo sandbox environment. It will be available for all soon.**
+:::
+
+::: tips
+This is an optional feature in Apps, you can also use your own Authentication.
+:::
+
+To authenticate the users coming from Akeneo PIM, you can use the OpenID Connect protocol.
+
+::: info
+**What is OpenID Connect?**  
+OpenID Connect is a simple identity layer on top of the OAuth 2.0 protocol.  
+See [OpenID official website](https://openid.net/connect/) for more info.
+:::
+
+Basically, you use the same process as for Authorization, but you request an additional scope
+and you will receive, alongside the Access Token, an ID Token containing the information of the current user.
+
+![App authentication diagram](../img/apps/app-authentication-sequence-diagram.png)
+
+## Authenticate during the first Authorization request
```

Comment:
```suggestion
## Authentication during the first Authorization request
```
and similar titles

Reply:
I will check the english with Camille, I'm expecting a review on the content from you ;)